simapp/sim_test.go

```diff
@@ -31,8 +31,14 @@ import (
 )
 
 func init() {
+	// TODO: remove "Simulation" prefix as it's verbose
 	flag.StringVar(&genesisFile, "SimulationGenesis", "", "custom simulation genesis file; cannot be used with params file")
 	flag.StringVar(&paramsFile, "SimulationParams", "", "custom simulation params file which overrides any random params; cannot be used with genesis")
+	flag.BoolVar(&exportParams, "SimulationExportParams", false, "export the randomly generated params to JSON at a export height")
```

Comment:
This seems redundant; The presence of a non-empty `exportParamsPath` should be enough.